source/draw/RasterBgr.ooc

```diff
@@ -110,7 +110,7 @@ RasterBgr: class extends RasterPacked {
 						result += (distance) sqrt() / 3
 					}
 				}
-			result /= ((this size width squared() + this size height squared()) as Float sqrt())
+			result /= this size length
```

Comment:
Don't you think this kind of changes should be done separately? This pull request is quite huge as it is, and this kind of changes probably adds a lot of time to the reviewing.